src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,38 @@
+import unpaidIcon from '../../assets/images/paymentrequest_status_unpaid.svg';
+import paidIcon from '../../assets/images/paymentrequest_status_paid.svg';
+import partiallyPaidIcon from '../../assets/images/paymentrequest_status_partial.svg';
+
+interface StatusBadgeProps {
+  status: 'UNPAID' | 'PARTIAL' | 'PAID';
+}
+
+const getIconForStatus = (status: 'UNPAID' | 'PARTIAL' | 'PAID') => {
+  switch (status) {
+    case 'PARTIAL':
+      return partiallyPaidIcon;
+    case 'PAID':
+      return paidIcon;
+    case 'UNPAID':
+    default:
+      return unpaidIcon;
+  }
+};
+
+const StatusBadge = ({ status }: StatusBadgeProps) => {
+  return (
+    <div
+      className={`rounded-full py-1 px-2 space-x-2 inline-block ${
+        status === 'PAID' ? 'bg-[#D8F2EA] text-[#008055]' : 'bg-[#F1F3F4] text-[#00264D]'
```

Comment:
Could you add the `#00264D` color to the as a tailwind color?

It seems that will be used in different places, and even in the Figma document the color was named "Default text" as it's quite used.